source/agora/test/Base.d

```diff
@@ -821,14 +826,15 @@ public struct TestConf
     Params:
         APIManager = Type of API manager to instantiate
         test_conf = the test configuration
+        params = the consensus-critical constants
 
     Returns:
         The set of public key added to the node
 
 *******************************************************************************/
 
 public APIManager makeTestNetwork (APIManager : TestAPIManager = TestAPIManager)(
-    in TestConf test_conf)
+    in TestConf test_conf, in Params params = null)
```

Comment:
`TestConf` is already supposed to carry all test-related configuration. So I suggest moving `Params` inside `TestConf`.

Reply:
> `TestConf` is already supposed to carry all test-related configuration. So I suggest moving `Params` inside `TestConf`.

I'll check your suggestion. Thanks.